modules/openapi-generator/src/main/resources/ruby-client/README.mustache

```diff
@@ -66,9 +66,12 @@ require '{{{gemName}}}'
 {{#apiInfo}}{{#apis}}{{#-first}}{{#operations}}{{#operation}}{{#-first}}{{#hasAuthMethods}}
 # Setup authorization
 {{{moduleName}}}.configure do |config|{{#authMethods}}{{#isBasic}}
+{{^isBasicBearer}}
```

Comment:
@bopm 
> I am struggle to find a way to remove empty lines in generated files.

How about changing the line break here?
```diff
$ git diff
diff --git a/modules/openapi-generator/src/main/resources/ruby-client/api_doc.mustache b/modules/openapi-generator/src/main/resources/ruby-client/api_doc.mustache
index ac7aff1d4f..dc163725e1 100644
--- a/modules/openapi-generator/src/main/resources/ruby-client/api_doc.mustache
+++ b/modules/openapi-generator/src/main/resources/ruby-client/api_doc.mustache
@@ -27,8 +27,7 @@ Method | HTTP request | Description
 require '{{{gemName}}}'
 {{#hasAuthMethods}}
 # setup authorization
-{{{moduleName}}}.configure do |config|{{#authMethods}}{{#isBasic}}
-{{^isBasicBearer}}
+{{{moduleName}}}.configure do |config|{{#authMethods}}{{#isBasic}}{{^isBasicBearer}}
   # Configure HTTP basic authorization: {{{name}}}
   config.username = 'YOUR USERNAME'
   config.password = 'YOUR PASSWORD'{{/isBasicBearer}}{{#isBasicBearer}}
diff --git a/samples/openapi3/client/petstore/ruby/docs/FakeApi.md b/samples/openapi3/client/petstore/ruby/docs/FakeApi.md
index 8c0dbc6989..ab79170143 100644
--- a/samples/openapi3/client/petstore/ruby/docs/FakeApi.md
+++ b/samples/openapi3/client/petstore/ruby/docs/FakeApi.md
@@ -547,7 +547,6 @@ Fake endpoint to test group parameters (optional)
 require 'petstore'
 # setup authorization
 Petstore.configure do |config|
-
   # Configure Bearer authorization (JWT): bearer_test
   config.access_token = 'YOUR_BEARER_TOKEN'
 end
```

Reply:
Seems like it did it. I guess I was looking on subset of changes last time I tried that yesterday. Anyway, it's fine now, thanks.